tests/unit/test_app.py

```
@@ -2430,17 +2383,15 @@ def test_autosave_opml_mocked(
     assert "http://example.com/autosave" in written_data
 
 
-def test_autosave_opml_with_temp_fs(tmp_path, client,
-                                    setup_autosave_test_data):
+def test_autosave_opml_with_temp_fs(tmp_path, client, setup_autosave_test_data):
```

Ruff (0.15.2)

[warning] Unused function argument: `setup_autosave_test_data`

(ARG001)